### PR TITLE
ImageAPI: Print `ProviderName`

### PR DESCRIPTION
**Pull request by ArdenHide**

### CODE REVIEW

File: src/ImageAPI/ProvidersImages/Advanced/RefundProviderImage.cs [hunk 1]
```diff
@@ -21,6 +22,9 @@ public class RefundProviderImage : ProviderImage
     public RefundProviderImage(Image backgroundImage, Font font, IReadOnlyList<DynamoDbItem> dynamoDbItems)
         : base(backgroundImage, font)
     {
-        Image = DrawAttributes(dynamoDbItems[0]);
+        Image = Image
+            .DrawBackgroundImage(backgroundImage)
+            .DrawProviderName(font, nameof(RefundProvider))
+            .DrawAttributes(font, dynamoDbItems[0], GetCoordinates);
```
Comment (Lomet) [inline src/ImageAPI/ProvidersImages/Advanced/RefundProviderImage.cs:28]: this code is duplicated across all code

File: src/ImageAPI/Utils/ImageExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
+﻿using SixLabors.Fonts;
+using SixLabors.ImageSharp;
+using MetaDataAPI.Models.DynamoDb;
+using SixLabors.ImageSharp.Processing;
+
+namespace ImageAPI.Utils;
+
+public static class ImageExtensions
```
Comment (Lomet) [inline src/ImageAPI/Utils/ImageExtensions.cs:8]: we can make it none-static, using fluent pattern.
all the function here take the same paraameters

File: src/ImageAPI/ProvidersImages/Simple/TimedDealProviderImage.cs [hunk 3]
```diff
@@ -1,15 +1,16 @@
...
 using SixLabors.ImageSharp;
+using MetaDataAPI.Providers;
 using MetaDataAPI.Models.DynamoDb;
 
 namespace ImageAPI.ProvidersImages.Simple;
 
 public class TimedDealProviderImage : ProviderImage
 {
-    public override Image Image { get; }
+    public sealed override Image Image { get; }
```
Comment (Lomet) [inline src/ImageAPI/ProvidersImages/Simple/TimedDealProviderImage.cs:11]: why to use `sealed`? better to unseal, and remove some code from the simple providers, got some duplicates there

**Review by Lomet — CHANGES_REQUESTED** [review]
good start

**Comment by ArdenHide** [comment]
[link](https://nft.poolz.finance/test/image?hash=148201df27690dad550628ad053af165c4a475cecff88af4f163e03c4a063c1f)

https://nft.poolz.finance/test/image?hash=148201df27690dad550628ad053af165c4a475cecff88af4f163e03c4a063c1f

**Review by Lomet — APPROVED** [review]

**Pull request merged**